### DISCUSSION TITLE
How to move the arrows in a song?

### DISCUSSION THREAD

**Discussion by demogamer** [Leather128/LeathersFunkinEngine#212] [Q&A]
I'm making an Applecore port of a song and I need to know how to change the position of the arrows in the song.

**Comment by sanamasako** [comment]
Songs that have unique gameplay features such as strumline movement have what we call "modcharts". They're coded in lua. Currently, LE uses Kade Engine's modchart system, so take a look [here](https://web.archive.org/web/20210420044709/https://github.com/KadeDev/Kade-Engine/blob/master/ModCharts.md) on how the basics of modcharts work. Definitely not enough for you to learn how to make modcharts off the top of your head, unless you're a god at figuring things out by yourself (but I mean, judging by the fact that this post is here I doubt it). I'd recommend looking up a tutorial on lua basics and then taking a look at the link I sent. It's in webarchive because apparently the modchart.md thing is gone, thank god someone archived it. Anyway, I hope this works or at least helps.
